internal/server/cluster/membership.go

```diff
@@ -14,7 +14,6 @@ import (
 
 	"github.com/cowsql/go-cowsql/app"
 	"github.com/cowsql/go-cowsql/client"
-
```

Comment:
Hmm, that one line change looks wrong, the line break is there to separate external and local imports.

Reply:
Ah, sorry, I guess this happened after running `gci` on my side.